src/components/Loader/index.tsx

```diff
@@ -2,11 +2,11 @@ import { CircularProgress, Grid } from '@mui/material';
 import styles from './Loader.module.css';
 
 const GordonLoader = ({ size = 100, color = 'primary', inline = false }) => {
-  if (inline) return <CircularProgress size={size} color={color} />;
+  if (inline) return <CircularProgress size={size} color={'primary'} />;
```

Comment:
This is a regression for the GordonLoader, since the color prop is now ignored and the loader's color is now just always 'primary'.

Reply:
Since you aren't declaring the types of the props for `GordonLoader`, typescript is doing its best to infer them. In this case, it infers the type of `color` to be string because as far as it's aware you can pass any kind of string in as the color. However, the `CircularProgress` component is much more strict about the type of it's `color` prop. So typescript is warning you that `GordonLoader`'s color could be any string whatsoever but `CircularProgress` only accepts a narrow subset of all possible strings, which could cause an error.

The fix, in this case, is to be explicit about the types of `GordonLoader`'s props. You can add a `Props` type like you have in most other components. Rather than trying to recreate the logic of `CircularProgress`'s props types, you can just use those props and add the `inline` prop that is specific to `GordonLoader`:
```typescript
  type Props = CircularProgressProps & {
      inline?: boolean
  }
```

Reply:
It worked, thank you!